db/queries/all.ts

```diff
@@ -0,0 +1,23 @@
+export const ALL_OUTGOING = `
```

Comment:
We should ideally keep SQL in .sql files rather than as long strings in TypeScript. I can take care of this later.